ext/rotoscope/extconf.rb

```diff
@@ -2,4 +2,6 @@
 require "mkmf"
 
 $CFLAGS << ' -std=c99 -Wall -Werror -Wno-declaration-after-statement'
+
+have_macro('_POSIX_SOURCE')
```

Comment:
It doesn't look like this does anything.  [have_macro](http://ruby-doc.org/stdlib-2.3.3/libdoc/mkmf/rdoc/MakeMakefile.html#method-i-have_macro) returns a value, but you aren't checking it and it doesn't document doing any side-effects